tests/aws/services/lambda_/test_lambda_api.py

```diff
@@ -5607,6 +5607,67 @@ def test_create_event_source_validation(
         response = e.value.response
         snapshot.match("error", response)
 
+    @markers.aws.validated
+    def test_create_event_filter_criteria_validation(
+        self,
+        create_lambda_function,
+        lambda_su_role,
+        dynamodb_create_table,
+        snapshot,
+        aws_client,
+    ):
+        function_name = f"function-{short_uid()}"
+        create_lambda_function(
+            handler_file=TEST_LAMBDA_PYTHON_ECHO,
+            func_name=function_name,
+            runtime=Runtime.python3_12,
+            role=lambda_su_role,
+        )
+
+        table_name = f"table-{short_uid()}"
+        # FIXME: Why is this not being automatically transformed?
```

Comment:
Do you expect DynamoDB transformers to apply automatically in the Lambda API tests?

Reply:
Look at the `test_event_source_mapping_lifecycle_delete_function`. It resides in the same class but has it's `create_response` correctly transformed.